src/Microsoft.DotNet.Interactive/Kernel.Static.cs

```diff
@@ -17,6 +17,8 @@ public partial class Kernel
     {
         public static Kernel Current => KernelInvocationContext.Current.HandlingKernel;
 
+        public static Kernel Root => Current.ParentKernel;
```

Comment:
This should walk all the way to the root in case of deeper hierarchies and return self in the case where the current kernel is the root.